### ISSUE TITLE
Pro: slug translation check fails if "post-formats" theme support is added without arguments

### ISSUE THREAD

**Issue by tnottu** [polylang/polylang#15]
If i add theme support for post-formats without arguments like this:
```
add_theme_support( 'post-formats' );
```
then `get_theme_support( 'post-formats' )` will return `true`. See here: [Codex: get theme support](https://codex.wordpress.org/Function_Reference/get_theme_support#Return)

If that is the case, a `foreach` loop in `modules/translate-slugs/translate-slugs-model.php` fails because it's trying to loop a boolean instead of an array.

```
// Line 189
// get_theme_support sends an array of array
if ( false !== $formats = get_theme_support( 'post-formats' ) ) {
	foreach ( $formats[0] as $format ) {
		$slugs[ 'post-format-' . $format ]['slug'] = $format;
		$slugs[ 'post-format-' . $format ]['translations'][ $language->slug ] = $mo->translate( $format );
	}
}
```

Instead of checking for `false`, perhaps the condition should check that `get_theme_support` returns an array?

I don't know if there's any reason to add support for post-formats without actually defining any, but this line was in a theme boilerplate I was using and caused a bit of confusion at first.

**Comment by Chouby** [comment]
I just tested `add_theme_support( 'post-formats' );` and for me it does not work as the format metabox is empty. But changing this won't do any harm :)

**Comment by tnottu** [comment]
I should have been more clear: the problem occured in the *String translations* admin view. With the conditions I mentioned, the string translations table shows a php error instead. I didn't actually test how the posts or metaboxes looked, I just removed the post-format support since i didn't need it :)

**Comment by Chouby** [comment]
The fix will be included in v1.9.3
